### PR TITLE
Feature/mui4

### PR DESCRIPTION
**Pull request by thiemok**
Upgrades mui to v4.
Required migrations are applied and some cosmetic changes to the dummyList are added.

### CODE REVIEW

File: client/containers/dummyList.jsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ import { SK5 } from 'components/spinkit';
 
 import type { List as ImmutableList } from 'immutable';
 
-const REFRESH_INTERVAL = 15000;
+const REFRESH_INTERVAL = 1500000000000000000000000;
```
Comment (maddrag0n) [inline client/containers/dummyList.jsx]: why the change in REFRESH_INTERVAL?

Comment (thiemok) [reply client/containers/dummyList.jsx]: Welp...

**Review by maddrag0n — DISMISSED** [review]

**Review by maddrag0n — APPROVED** [review]
